tsl/test/sql/policy_generalization.sql

```diff
@@ -28,4 +28,38 @@ select count(*) from timescaledb_information.chunks where hypertable_name='test'
 SELECT count(*) from drop_chunks('test', created_after => INTERVAL '1 hour');
 select count(*) from timescaledb_information.chunks where hypertable_name='test';
 
+-- retention policy
+INSERT INTO test SELECT i, i %10, 0.10 FROM generate_series(1, 100, 1) i;
+\set ON_ERROR_STOP 0
+-- interval input for "drop_after" for INTEGER partitioning errors out
+SELECT add_retention_policy('test', INTERVAL '5 seconds', true);
```

Comment:
Test that trying to add a retention policy with both created_at and created_before will fail?

Reply:
Added